mycore-base/src/main/java/org/mycore/frontend/cli/MCRCommandLineInterface.java

```
@@ -321,19 +325,35 @@ public static void getURI(String uri, String file) throws Exception {
     public static List<String> readCommandsFile(String file) throws IOException {
         try (BufferedReader reader = Files.newBufferedReader(new File(file).toPath(), Charset.defaultCharset())) {
             output("Reading commands from file " + file);
+            return readCommandsFromBufferedReader(reader);
+        }
+    }
+
+    public static List<String> readCommandsRessource(String resource) throws IOException {
+        final URL resourceURL = MCRClassTools.getClassLoader().getResource(resource);
+        if (resourceURL == null) {
+            return Collections.emptyList();
```

Should throw an IOException here if the resource is not found.